### ISSUE TITLE
sumatrapdf as TotalCMD-x64 plugin crashed while copying text from PDF

### ISSUE THREAD

**Issue by den1s0v** [sumatrapdfreader/sumatrapdf#3452]
Crash log as gist:
[https://gist.github.com/den1s0v/e6de1328b10ea7bcf1e122a0d170656a](https://gist.github.com/den1s0v/e6de1328b10ea7bcf1e122a0d170656a)  
Dump file generated in `Total Commander\Plugins\wlx\slister\crashinfo\`: [sumatrapdfcrash.dmp.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/11288403/sumatrapdfcrash.dmp.zip)  
The error appeared with PDF : [2022_Software_Impacts_CorrectWriting_Revised.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/11288407/2022_Software_Impacts_CorrectWriting_Revised.pdf)

Steps I did to reproduce:
- open the PDF above via TotalCommander's F3 (inner viewer set up as SumatraPDF plugin for PDFs).
- the file reads, shows and scrolls fine, the text can be mouse-selected.
- I opened the file and it was opened for several hours, and PC could go to sleep mode once or twice.
- then I was back to reading and selected some text with mouse (this piece, to be exact: `Typo analyzer also checks for omitted separators (merges two tokens into one) and extraneous  separator (splits one token into two).`)
- Then I rapidly hit `Ctrl+С` twice (I copy text twice in many software since some do not set clipboard at the first try.)
- Then, shortly, I saw error message report (next, I went to reporting it here [as developers asked](https://www.sumatrapdfreader.org/docs/Submit-crash-report)).
- Interestingly, the copied text has managed to reach the clipboard (I have clipboard-tracking utility enabled - Punto Switcher, so I can see that). So I think that crash was caused by my second hit of `Ctrl+C`.

P.S. _Thank you for this fine PDF reader!_

**Comment by GitHubRulesOK** [comment]
hmm looks reasonable HOWEVER the debug info for that version 3.5.15304 is no longer available as we are now at 3.5.15481 so perhaps save that version just in case there are ther issues but update to current pre-release https://www.sumatrapdfreader.org/prerelease and fist task select download symbols for next crash :-) 
![image](https://user-images.githubusercontent.com/12754913/233439358-b11db952-6bb5-4d3b-8501-88e7ca333842.png)

**Comment by den1s0v** [comment]
Thank you for the hint.
It **works fine now**, both standalone and as plugin from within Total Commander, with the latest pre-release I downloaded from you link (build version 15481, built on 2023-04-18, commit [9a2183db3c3db5cbf242ac9d8f8576750f581096](https://github.com/sumatrapdfreader/sumatrapdf/commit/9a2183db3c3db5cbf242ac9d8f8576750f581096))

So the issue seems to be resolved.

I have experiments with older version I used, and it crashed only as TC plugin, but OK when run as standalone. This older version still can copy without an error if using `right-click > Copy selection`.
